frontend/src/containers/NavigationTree.js

```diff
@@ -149,7 +145,7 @@ class NavigationTree extends Component {
     const input = document.getElementById('search-input');
     const firstMenuItem = document.getElementsByClassName('js-menu-item')[0];
     let prevParentSibling = document.activeElement.previousSibling;
-
+    const elementToEnter = document.activeElement.childNodes[0];
```

Comment:
why not having it at line 178, right were is used?

Reply:
eslint would complain that we have def within case blocks